app/src/main/java/nu/yona/app/ui/YonaActivity.java

```diff
@@ -342,9 +343,31 @@ public void onResume()
 			{
 				checkVPN();
 			}
+			checkForNotificationPermission();
 		}
 	}
 
+	private void checkForNotificationPermission()
+	{
+		if (!NotificationManagerCompat.from(this).areNotificationsEnabled())
+		{
+			Logger.loge("Notifications Disabled", this.getString(R.string.notification_disabled_message));
+			displayErrorAlert(new ErrorMessage(this.getString(R.string.notification_disabled_message)));
+		}
+	}
+
+	public static void displayErrorAlert(ErrorMessage errorMessage)
```

Comment:
Why are you pulling this one and the next one out of ``AppUtils.java``? These methods are generic utility methods, so I favor leaving them there.